packages/eas-cli/src/credentials/ios/appstore/ensureAppExists.ts

```diff
@@ -72,23 +72,33 @@ export async function ensureBundleIdExistsWithNameAsync(
   }
 
   if (options) {
-    try {
-      spinner = ora(`Syncing capabilities`).start();
+    await syncCapabilities(bundleId, options);
+  }
+}
 
-      // Update the capabilities
+export async function syncCapabilities(
+  bundleId: BundleId,
+  options: EnsureAppExistsOptions
+): Promise<void> {
+  const spinner = ora(`Syncing capabilities`).start();
+
+  try {
+    const capability = await bundleId.hasCapabilityAsync(CapabilityType.PUSH_NOTIFICATIONS);
```

Comment:
Seems fine, but this won't scale well if we add other capabilities in the future.